spec/models/invite_status_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe InviteStatus, type: :model do
+  subject { InviteStatus }
+
```

Comment:
In addition to these tests, could you also add tests ensuring the ActiveModel relationships are working correctly?

We might also want tests in the user model spec asserting we can do `user.assignment_invitations` and that works correctly.

Also a test showing that if a user gets destroyed, it's invitestatuses get destroyed, but the assignment invitation itself does _not_ get destroyed would be great.